Bonsai.Core/Expressions/IncludeWorkflowBuilder.cs

```diff
@@ -24,6 +24,8 @@ namespace Bonsai.Expressions
     public sealed class IncludeWorkflowBuilder : VariableArgumentExpressionBuilder, IGroupWorkflowBuilder, INamedElement, IRequireBuildContext
     {
         const char AssemblySeparator = ':';
+        internal const string BuildUriPrefix = "::build:";
+        const string DefaultSerializerPropertyName = "Property";
```

Comment:
I've gathered that this is part of what allows using the same `XmlSerializer` for the same types in multiple unrelated contexts, but it took a while to get there.

Would `Placeholder` maybe be better than `Default` here? A clarifying comment that this is only used ephemerally to allow sharing `XmlSerializer` instances might be helpful too.